-- MySQL dump 10.13  Distrib 5.7.19, for osx10.12 (x86_64)
--
-- Host: mysql-prd2.broadinstitute.org    Database: DSPRegressionTesting
-- ------------------------------------------------------
-- Server version	5.7.17-log

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8 */;
/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
/*!40103 SET TIME_ZONE='+00:00' */;
/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;


USE DSPRegressionTesting;

--
-- Table structure for table `AnalysisInfo`
--

DROP TABLE IF EXISTS `AnalysisInfo`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `AnalysisInfo` (
  `idAnalysisInfo` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `version` varchar(1024) COLLATE utf8mb4_unicode_ci NOT NULL,
  `name` varchar(512) COLLATE utf8mb4_unicode_ci NOT NULL,
  `wdl` varchar(1024) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `wdlChecksum` char(32) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  PRIMARY KEY (`idAnalysisInfo`),
  UNIQUE KEY `idAnalysisInfo_UNIQUE` (`idAnalysisInfo`)
) ENGINE=InnoDB AUTO_INCREMENT=4 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `AnalysisInfo`
--

LOCK TABLES `AnalysisInfo` WRITE;
/*!40000 ALTER TABLE `AnalysisInfo` DISABLE KEYS */;
INSERT INTO `AnalysisInfo` VALUES (1,'broadinstitute/gatk-nightly:2019-02-26-4.1.0.0-31-g23bd0a2f8-SNAPSHOT','Concordance','gs://broad-dsp-methods-regression-testing/analysisWdls/variantCallerConcordance.wdl','27bdf03e20eb05e7de0b6253ba99d6a0'),(2,'broadinstitute/gatk-nightly:2019-02-26-4.1.0.0-31-g23bd0a2f8-SNAPSHOT','Genotype Concordance','gs://broad-dsp-methods-regression-testing/analysisWdls/genotypeConcordance.wdl','f2951a4484c6a7720dc3a698940d2ccf'),(3,'broadinstitute/gatk-nightly:2019-02-26-4.1.0.0-31-g23bd0a2f8-SNAPSHOT','Timing',NULL,NULL);
/*!40000 ALTER TABLE `AnalysisInfo` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `AnalysisOutputFiles`
--

DROP TABLE IF EXISTS `AnalysisOutputFiles`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `AnalysisOutputFiles` (
  `analysis` int(11) unsigned NOT NULL,
  `outputFile` int(11) unsigned NOT NULL,
  PRIMARY KEY (`analysis`,`outputFile`),
  KEY `AnalysisOutputFiles_outputFileID_idx` (`outputFile`),
  CONSTRAINT `AnalysisOutputFiles_analysisRunID` FOREIGN KEY (`analysis`) REFERENCES `AnalysisRuns` (`idAnalysisRun`) ON DELETE NO ACTION ON UPDATE NO ACTION,
  CONSTRAINT `AnalysisOutputFiles_outputFileID` FOREIGN KEY (`outputFile`) REFERENCES `OutputFiles` (`idOutputFiles`) ON DELETE NO ACTION ON UPDATE NO ACTION
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `AnalysisOutputFiles`
--

LOCK TABLES `AnalysisOutputFiles` WRITE;
/*!40000 ALTER TABLE `AnalysisOutputFiles` DISABLE KEYS */;
/*!40000 ALTER TABLE `AnalysisOutputFiles` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `AnalysisRuns`
--

DROP TABLE IF EXISTS `AnalysisRuns`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `AnalysisRuns` (
  `idAnalysisRun` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `analysisInfoID` int(11) unsigned NOT NULL,
  `scenarioOutputForComparison` int(11) unsigned DEFAULT NULL,
  `configuration` json DEFAULT NULL,
  PRIMARY KEY (`idAnalysisRun`),
  UNIQUE KEY `idAnalysis_UNIQUE` (`idAnalysisRun`),
  KEY `Analysis_scenarioOutputForComparison_idx` (`scenarioOutputForComparison`),
  KEY `AnalysisRuns_analysisInfoID_idx` (`analysisInfoID`),
  CONSTRAINT `AnalysisRuns_analysisInfoID` FOREIGN KEY (`analysisInfoID`) REFERENCES `AnalysisInfo` (`idAnalysisInfo`) ON DELETE NO ACTION ON UPDATE NO ACTION,
  CONSTRAINT `AnalysisRuns_scenarioOutputForComparison` FOREIGN KEY (`scenarioOutputForComparison`) REFERENCES `OutputFiles` (`idOutputFiles`) ON DELETE NO ACTION ON UPDATE NO ACTION
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `AnalysisRuns`
--

LOCK TABLES `AnalysisRuns` WRITE;
/*!40000 ALTER TABLE `AnalysisRuns` DISABLE KEYS */;
/*!40000 ALTER TABLE `AnalysisRuns` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `BaselineInputFiles`
--

DROP TABLE IF EXISTS `BaselineInputFiles`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `BaselineInputFiles` (
  `idBaselineInputFiles` int(11) unsigned NOT NULL AUTO_INCREMENT COMMENT 'Unique ID of each file.',
  `path` varchar(2048) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'Path to the file.',
  `type` varchar(45) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'Type of each file.',
  `md5sum` char(32) COLLATE utf8mb4_unicode_ci NOT NULL,
  PRIMARY KEY (`idBaselineInputFiles`),
  UNIQUE KEY `idInputFiles_UNIQUE` (`idBaselineInputFiles`),
  UNIQUE KEY `BaselineInputFiles_md5sum_UNIQUE` (`md5sum`),
  KEY `BaselineInputFiles_fileType_idx` (`type`),
  CONSTRAINT `InputFiles_fileType` FOREIGN KEY (`type`) REFERENCES `FileTypes` (`type`) ON DELETE NO ACTION ON UPDATE NO ACTION
) ENGINE=InnoDB AUTO_INCREMENT=42 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `BaselineInputFiles`
--

LOCK TABLES `BaselineInputFiles` WRITE;
/*!40000 ALTER TABLE `BaselineInputFiles` DISABLE KEYS */;
INSERT INTO `BaselineInputFiles` VALUES (1,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-359781.bai','EXOME_READS_INDEX','66fb4afaeb9bcf1fb6ba0c47e302925b'),(2,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-359781.bam','EXOME_READS','2736eb455d31ede2591aae8216fb6eda'),(3,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-359781.vcf.gz','SOMATIC_VCF','dcec297c7bcd6f3f84566960a27c8b39'),(4,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-359781.vcf.gz.tbi','SOMATIC_VCF_INDEX','d78c8b92abb674356f24e20949dfeba8'),(5,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-359877.bai','EXOME_READS_INDEX','19109d326717b2f16ba4f426483b9a05'),(6,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-359877.bam','EXOME_READS','c9308aed697ef83cb8bfae623cf129f4'),(7,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-359877.vcf.gz','SOMATIC_VCF','d3e640c0c89a34ac7c984ea1772a5b9d'),(8,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-359877.vcf.gz.tbi','SOMATIC_VCF_INDEX','40dc97fe61be9d63b6b7cd9634e451bf'),(9,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-360361.bai','EXOME_READS_INDEX','0ac2525ec8f721e41a07517531b43162'),(10,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-360361.bam','EXOME_READS','34e3ac44bbf74b4012b5c9dc6ac1870d'),(11,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-360361.vcf.gz','SOMATIC_VCF','9c048ce896f1d80923cc1513ce29b137'),(12,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-360361.vcf.gz.tbi','SOMATIC_VCF_INDEX','5e2190bf1c9bdbf8ffafcf31da99ea20'),(13,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-360457.bai','EXOME_READS_INDEX','36e3ff23b02f2e308ebf90dcdeb902d9'),(14,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-360457.bam','EXOME_READS','3646f9c52ab317ee2bc909e28f273f9b'),(15,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-360457.vcf.gz','SOMATIC_VCF','5882c66818f7d5fe09104c485621775b'),(16,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-360457.vcf.gz.tbi','SOMATIC_VCF_INDEX','221fdceeebfa5edaefac873d8cc1905c'),(17,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-361337.bai','EXOME_READS_INDEX','5d8d45855f73b8d066706fa1f211b2b6'),(18,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-361337.bam','EXOME_READS','077d370374b2821defc1fb5d3f85966d'),(19,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-361337.vcf.gz','SOMATIC_VCF','7e5f03272f76f9b18f64b6609506bc76'),(20,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-361337.vcf.gz.tbi','SOMATIC_VCF_INDEX','395408a6c770d20255cb5b11d4ab35e7'),(21,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-361433.bai','EXOME_READS_INDEX','20f2438239dffa9673d35dff58b1381d'),(22,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-361433.bam','EXOME_READS','cc022a43a5454ab956a65a164bc047e4'),(23,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-361433.vcf.gz','SOMATIC_VCF','658583c40648a89a527fe161f5341e7b'),(24,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-361433.vcf.gz.tbi','SOMATIC_VCF_INDEX','ec7a13fea727be1636d38e1af8149d94'),(25,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-362428.bai','EXOME_READS_INDEX','53e700aafd1bc7bddd9a7bc4d4f937e7'),(26,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-362428.bam','EXOME_READS','5a6efabedc3a808ef487429031c8786c'),(27,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-362428.vcf.gz','SOMATIC_VCF','8f0afe2ec90d038271e41d543b34ee7a'),(28,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-362428.vcf.gz.tbi','SOMATIC_VCF_INDEX','40649c2f4baba87710901f1f112fc985'),(29,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-363907.bai','EXOME_READS_INDEX','70578295ea56edd1ce2a62ba41959e6d'),(30,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-363907.bam','EXOME_READS','bb4c81559e3002fcdd09ac48325aecbb'),(31,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-363907.vcf.gz','SOMATIC_VCF','c93f8294c25d1965a810a16bd500be95'),(32,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-363907.vcf.gz.tbi','SOMATIC_VCF_INDEX','7edba1f10be111e128ac726c04dd2ff7'),(33,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-445394.bai','EXOME_READS_INDEX','e4e3336d659b215a91cb81a1a18f4bf9'),(34,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-445394.bam','EXOME_READS','583aa5f341992912221ed3091ff1daf6'),(35,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-445394.vcf.gz','SOMATIC_VCF','880223730fa30dcb60801440bb6a226a'),(36,'gs://broad-dsp-methods-regression-testing/inputData/NexPond-445394.vcf.gz.tbi','SOMATIC_VCF_INDEX','0440125418ac9e3f601b65ad674f0727'),(37,'gs://broad-dsp-methods-regression-testing/inputData/G96830.NA12878.bam','GENOME_READS','7bb97f2a2c6ac23a52ab71d0175d897e'),(38,'gs://broad-dsp-methods-regression-testing/inputData/G96830.NA12878.bai','GENOME_READS_INDEX','9cb49ac1335443baa381e777dd9f485a'),(39,'gs://broad-dsp-methods-regression-testing/inputData/G96830.NA12878.vcf.gz','SOMATIC_VCF','ee32e9a04311032539e16c3ad41a4ea1'),(40,'gs://broad-dsp-methods-regression-testing/inputData/G96830.NA12878.vcf.gz.tbi','SOMATIC_VCF_INDEX','f3bb1d7aa99b9dfa7d82cfd7c3cf39be');
/*!40000 ALTER TABLE `BaselineInputFiles` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `FileTypes`
--

DROP TABLE IF EXISTS `FileTypes`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `FileTypes` (
  `idFileTypes` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `type` varchar(45) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The type of the file.',
  PRIMARY KEY (`idFileTypes`,`type`),
  UNIQUE KEY `idFileTypes_UNIQUE` (`idFileTypes`),
  UNIQUE KEY `FileTypes_type_UNIQUE` (`type`)
) ENGINE=InnoDB AUTO_INCREMENT=10 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `FileTypes`
--

LOCK TABLES `FileTypes` WRITE;
/*!40000 ALTER TABLE `FileTypes` DISABLE KEYS */;
INSERT INTO `FileTypes` VALUES (1,'EXOME_READS'),(2,'GENOME_READS'),(3,'SOMATIC_VCF'),(4,'GERMLINE_VCF'),(5,'EXOME_READS_INDEX'),(6,'GENOME_READS_INDEX'),(7,'SOMATIC_VCF_INDEX'),(8,'GERMLINE_VCF_INDEX'),(9,'UNSTRUCTURED_TEXT');
/*!40000 ALTER TABLE `FileTypes` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `Metric_Concordance_FilterAnalysis`
--

DROP TABLE IF EXISTS `Metric_Concordance_FilterAnalysis`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `Metric_Concordance_FilterAnalysis` (
  `idConcordance_FilterAnalysis` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `filter` varchar(64) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `tn` int(11) DEFAULT NULL,
  `fn` int(11) DEFAULT NULL,
  `uniqueTn` int(11) DEFAULT NULL,
  `uniqueFn` int(11) DEFAULT NULL,
  PRIMARY KEY (`idConcordance_FilterAnalysis`),
  UNIQUE KEY `idConcordance_FilterAnalysis_UNIQUE` (`idConcordance_FilterAnalysis`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `Metric_Concordance_FilterAnalysis`
--

LOCK TABLES `Metric_Concordance_FilterAnalysis` WRITE;
/*!40000 ALTER TABLE `Metric_Concordance_FilterAnalysis` DISABLE KEYS */;
/*!40000 ALTER TABLE `Metric_Concordance_FilterAnalysis` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `Metric_Concordance_Summary`
--

DROP TABLE IF EXISTS `Metric_Concordance_Summary`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `Metric_Concordance_Summary` (
  `idConcordance_Summary` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `variantType` varchar(512) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `truePositive` int(11) DEFAULT NULL,
  `falsePositive` int(11) DEFAULT NULL,
  `falseNegative` int(11) DEFAULT NULL,
  `concordanceSensitivity` double DEFAULT NULL,
  `concordancePrecision` double DEFAULT NULL,
  PRIMARY KEY (`idConcordance_Summary`),
  UNIQUE KEY `idConcordance_Summary_UNIQUE` (`idConcordance_Summary`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `Metric_Concordance_Summary`
--

LOCK TABLES `Metric_Concordance_Summary` WRITE;
/*!40000 ALTER TABLE `Metric_Concordance_Summary` DISABLE KEYS */;
/*!40000 ALTER TABLE `Metric_Concordance_Summary` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `Metric_GenotypeConcordance_ContingencyMetrics`
--

DROP TABLE IF EXISTS `Metric_GenotypeConcordance_ContingencyMetrics`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `Metric_GenotypeConcordance_ContingencyMetrics` (
  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `variantType` varchar(256) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `truthSample` varchar(256) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `callSample` varchar(256) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `tpCount` int(11) DEFAULT NULL,
  `tnCount` int(11) DEFAULT NULL,
  `fpCount` int(11) DEFAULT NULL,
  `fnCount` int(11) DEFAULT NULL,
  `emptyCount` int(11) DEFAULT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `id_metric_GenotypeConcordance_ContingencyMetrics_UNIQUE` (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `Metric_GenotypeConcordance_ContingencyMetrics`
--

LOCK TABLES `Metric_GenotypeConcordance_ContingencyMetrics` WRITE;
/*!40000 ALTER TABLE `Metric_GenotypeConcordance_ContingencyMetrics` DISABLE KEYS */;
/*!40000 ALTER TABLE `Metric_GenotypeConcordance_ContingencyMetrics` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `Metric_GenotypeConcordance_DetailMetrics`
--

DROP TABLE IF EXISTS `Metric_GenotypeConcordance_DetailMetrics`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `Metric_GenotypeConcordance_DetailMetrics` (
  `idGenotypeConcordance_DetailMetrics` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `variantType` varchar(512) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `truthSample` varchar(512) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `callSample` varchar(512) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `truthState` varchar(512) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `callState` varchar(512) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `count` int(11) DEFAULT NULL,
  `contingencyValues` varchar(512) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  PRIMARY KEY (`idGenotypeConcordance_DetailMetrics`),
  UNIQUE KEY `idGenotypeConcordance_DetailMetrics_UNIQUE` (`idGenotypeConcordance_DetailMetrics`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `Metric_GenotypeConcordance_DetailMetrics`
--

LOCK TABLES `Metric_GenotypeConcordance_DetailMetrics` WRITE;
/*!40000 ALTER TABLE `Metric_GenotypeConcordance_DetailMetrics` DISABLE KEYS */;
/*!40000 ALTER TABLE `Metric_GenotypeConcordance_DetailMetrics` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `Metric_GenotypeConcordance_SummaryMetrics`
--

DROP TABLE IF EXISTS `Metric_GenotypeConcordance_SummaryMetrics`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `Metric_GenotypeConcordance_SummaryMetrics` (
  `idGenotypeConcordance_SummaryMetrics` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `variantType` varchar(512) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `truthSample` varchar(512) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `callSample` varchar(512) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `hetSensitivity` double DEFAULT NULL,
  `hetPPV` double DEFAULT NULL,
  `hetSpecificity` double DEFAULT NULL,
  `homvarSensitivity` double DEFAULT NULL,
  `homvarPPV` double DEFAULT NULL,
  `homvarSpecificity` double DEFAULT NULL,
  `varSensitivity` double DEFAULT NULL,
  `varPPV` double DEFAULT NULL,
  `varSpecificity` double DEFAULT NULL,
  `genotypeConcordance` double DEFAULT NULL,
  `nonRefGenotypeConcordance` double DEFAULT NULL,
  PRIMARY KEY (`idGenotypeConcordance_SummaryMetrics`),
  UNIQUE KEY `idGenotypeConcordance_SummaryMetrics_UNIQUE` (`idGenotypeConcordance_SummaryMetrics`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `Metric_GenotypeConcordance_SummaryMetrics`
--

LOCK TABLES `Metric_GenotypeConcordance_SummaryMetrics` WRITE;
/*!40000 ALTER TABLE `Metric_GenotypeConcordance_SummaryMetrics` DISABLE KEYS */;
/*!40000 ALTER TABLE `Metric_GenotypeConcordance_SummaryMetrics` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `Metric_Timing`
--

DROP TABLE IF EXISTS `Metric_Timing`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `Metric_Timing` (
  `idMetric_Timing` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `startTime` double DEFAULT NULL COMMENT 'Epoch start time of this tool.',
  `endTime` double DEFAULT NULL COMMENT 'Epoch end time of this tool.',
  `elapsedTime` double DEFAULT NULL COMMENT 'Elapsed time of this tool (seconds).',
  PRIMARY KEY (`idMetric_Timing`),
  UNIQUE KEY `idMetric_Timing_UNIQUE` (`idMetric_Timing`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `Metric_Timing`
--

LOCK TABLES `Metric_Timing` WRITE;
/*!40000 ALTER TABLE `Metric_Timing` DISABLE KEYS */;
/*!40000 ALTER TABLE `Metric_Timing` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `Metrics`
--

DROP TABLE IF EXISTS `Metrics`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `Metrics` (
  `idMetrics` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `metricTableName` varchar(1024) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The name of the metric table containing the information.  Also the name of the metric collected.',
  `concreteMetricID` int(11) NOT NULL,
  `sourceAnalysis` int(11) unsigned NOT NULL,
  PRIMARY KEY (`idMetrics`),
  UNIQUE KEY `idMetrics_UNIQUE` (`idMetrics`),
  KEY `Metrics_sourceAnalysisRunID` (`sourceAnalysis`),
  CONSTRAINT `Metrics_sourceAnalysisRunID` FOREIGN KEY (`sourceAnalysis`) REFERENCES `AnalysisRuns` (`idAnalysisRun`) ON DELETE NO ACTION ON UPDATE NO ACTION
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `Metrics`
--

LOCK TABLES `Metrics` WRITE;
/*!40000 ALTER TABLE `Metrics` DISABLE KEYS */;
/*!40000 ALTER TABLE `Metrics` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `MetricsEvaluator`
--

DROP TABLE IF EXISTS `MetricsEvaluator`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `MetricsEvaluator` (
  `idMetricsEvaluator` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `scenarioInfo` int(11) unsigned NOT NULL,
  `metric` int(11) unsigned NOT NULL,
  `absoluteAllowableMaxValue` double DEFAULT NULL,
  `absoluteAllowableMinValue` double DEFAULT NULL,
  `pairwiseAllowableIncrease` double DEFAULT NULL,
  `pairwiseAllowableDecrease` double DEFAULT NULL,
  `isPairwiseMeasurePercentage` tinyint(4) DEFAULT NULL,
  PRIMARY KEY (`idMetricsEvaluator`),
  UNIQUE KEY `idMetricsEvaluator_UNIQUE` (`idMetricsEvaluator`),
  KEY `MetricsEvaluator_scenarioInfoID_idx` (`scenarioInfo`),
  KEY `MetricsEvaluator_metricID
_idx` (`metric`),
  CONSTRAINT `MetricsEvaluator_metricID
` FOREIGN KEY (`metric`) REFERENCES `Metrics` (`idMetrics`) ON DELETE NO ACTION ON UPDATE NO ACTION,
  CONSTRAINT `MetricsEvaluator_scenarioInfoID` FOREIGN KEY (`scenarioInfo`) REFERENCES `TestScenarioInfo` (`idTestScenarioInfo`) ON DELETE NO ACTION ON UPDATE NO ACTION
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `MetricsEvaluator`
--

LOCK TABLES `MetricsEvaluator` WRITE;
/*!40000 ALTER TABLE `MetricsEvaluator` DISABLE KEYS */;
/*!40000 ALTER TABLE `MetricsEvaluator` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `MetricsEvaluatorData`
--

DROP TABLE IF EXISTS `MetricsEvaluatorData`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `MetricsEvaluatorData` (
  `metricsEvaluatorID` int(11) unsigned NOT NULL,
  `dataID` int(11) unsigned NOT NULL,
  PRIMARY KEY (`metricsEvaluatorID`,`dataID`),
  KEY `MetricsEvaluatorData_dataID_idx` (`dataID`),
  CONSTRAINT `MetricsEvaluatorData_dataID` FOREIGN KEY (`dataID`) REFERENCES `OutputFiles` (`idOutputFiles`) ON DELETE NO ACTION ON UPDATE NO ACTION,
  CONSTRAINT `MetricsEvaluatorData_metricsEvaluatorID` FOREIGN KEY (`metricsEvaluatorID`) REFERENCES `MetricsEvaluator` (`idMetricsEvaluator`) ON DELETE NO ACTION ON UPDATE NO ACTION
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `MetricsEvaluatorData`
--

LOCK TABLES `MetricsEvaluatorData` WRITE;
/*!40000 ALTER TABLE `MetricsEvaluatorData` DISABLE KEYS */;
/*!40000 ALTER TABLE `MetricsEvaluatorData` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `OutputFiles`
--

DROP TABLE IF EXISTS `OutputFiles`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `OutputFiles` (
  `idOutputFiles` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `fileType` varchar(45) COLLATE utf8mb4_unicode_ci NOT NULL,
  `path` varchar(2048) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `timeCreated` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
  `md5sum` char(32) COLLATE utf8mb4_unicode_ci NOT NULL,
  `sourceType` enum('TOOL','ANALYSIS') COLLATE utf8mb4_unicode_ci NOT NULL,
  PRIMARY KEY (`idOutputFiles`),
  UNIQUE KEY `idOutputFiles_UNIQUE` (`idOutputFiles`),
  KEY `OutputFiles_fileType_idx` (`fileType`),
  CONSTRAINT `OutputFiles_fileType` FOREIGN KEY (`fileType`) REFERENCES `FileTypes` (`type`) ON DELETE NO ACTION ON UPDATE NO ACTION
) ENGINE=InnoDB AUTO_INCREMENT=199 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `OutputFiles`
--

LOCK TABLES `OutputFiles` WRITE;
/*!40000 ALTER TABLE `OutputFiles` DISABLE KEYS */;
INSERT INTO `OutputFiles` VALUES (1,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/84edd0ed-f084-47bb-af11-d8b47b9f1865/call-HaplotypeCallerTask/shard-0/NexPond-359781.HC.vcf','2019-03-13 00:29:18','4ba0408ce4772d4d17400d1e69bbbc84','TOOL'),(2,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/84edd0ed-f084-47bb-af11-d8b47b9f1865/call-HaplotypeCallerTask/shard-0/NexPond-359781.HC.vcf.idx','2019-03-13 00:29:14','d03b2a37687f3fdc3b6ffdecba4e9deb','TOOL'),(3,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/84edd0ed-f084-47bb-af11-d8b47b9f1865/call-HaplotypeCallerTask/shard-1/NexPond-359877.HC.vcf','2019-03-13 00:29:18','5bdb0ab6bfb10c0e3bbbedb01c3e9bf2','TOOL'),(4,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/84edd0ed-f084-47bb-af11-d8b47b9f1865/call-HaplotypeCallerTask/shard-1/NexPond-359877.HC.vcf.idx','2019-03-13 00:29:18','7f5d8402bfdf2538164f53307ddd8a3e','TOOL'),(5,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/84edd0ed-f084-47bb-af11-d8b47b9f1865/call-HaplotypeCallerTask/shard-2/NexPond-360361.HC.vcf','2019-03-13 00:29:21','e11a6b8db01780f43d7a299617989885','TOOL'),(6,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/84edd0ed-f084-47bb-af11-d8b47b9f1865/call-HaplotypeCallerTask/shard-2/NexPond-360361.HC.vcf.idx','2019-03-13 00:29:18','f22efd59e6eaab0cc0fbef351c7984e9','TOOL'),(7,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/84edd0ed-f084-47bb-af11-d8b47b9f1865/call-HaplotypeCallerTask/shard-3/NexPond-360457.HC.vcf','2019-03-13 00:29:19','bbd66b220d31b66af2ec05a8b96cb0d6','TOOL'),(8,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/84edd0ed-f084-47bb-af11-d8b47b9f1865/call-HaplotypeCallerTask/shard-3/NexPond-360457.HC.vcf.idx','2019-03-13 00:29:22','d7739f9156ed18767bd780b489e512e8','TOOL'),(9,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/84edd0ed-f084-47bb-af11-d8b47b9f1865/call-HaplotypeCallerTask/shard-4/NexPond-361337.HC.vcf','2019-03-13 00:29:18','621be91f8c34d87d0e51c03c2e13ecf3','TOOL'),(10,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/84edd0ed-f084-47bb-af11-d8b47b9f1865/call-HaplotypeCallerTask/shard-4/NexPond-361337.HC.vcf.idx','2019-03-13 00:29:18','a7fcda4bc8ca1149a8a41714f42c6231','TOOL'),(11,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/84edd0ed-f084-47bb-af11-d8b47b9f1865/call-HaplotypeCallerTask/shard-5/NexPond-361433.HC.vcf','2019-03-13 00:29:19','11853c27a13ac0a0a9fa994fca31831c','TOOL'),(12,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/84edd0ed-f084-47bb-af11-d8b47b9f1865/call-HaplotypeCallerTask/shard-5/NexPond-361433.HC.vcf.idx','2019-03-13 00:29:18','a2dad6361325d6ffa23992b7188fb706','TOOL'),(13,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/84edd0ed-f084-47bb-af11-d8b47b9f1865/call-HaplotypeCallerTask/shard-6/NexPond-362428.HC.vcf','2019-03-13 00:29:20','3813c36152de6e0b4c29b6ddc3c39e7c','TOOL'),(14,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/84edd0ed-f084-47bb-af11-d8b47b9f1865/call-HaplotypeCallerTask/shard-6/NexPond-362428.HC.vcf.idx','2019-03-13 00:29:19','88db43e3129f5fc9c7c1b3cdc8b1e3aa','TOOL'),(15,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/84edd0ed-f084-47bb-af11-d8b47b9f1865/call-HaplotypeCallerTask/shard-7/NexPond-363907.HC.vcf','2019-03-13 00:29:18','d0465890be01ccead62fbff2aeb88395','TOOL'),(16,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/84edd0ed-f084-47bb-af11-d8b47b9f1865/call-HaplotypeCallerTask/shard-7/NexPond-363907.HC.vcf.idx','2019-03-13 00:29:20','9a66e7cb83ae43664bd93b220f60bd01','TOOL'),(17,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/84edd0ed-f084-47bb-af11-d8b47b9f1865/call-HaplotypeCallerTask/shard-8/NexPond-445394.HC.vcf','2019-03-13 00:29:18','5ca98b4ca09ca42da7ff47ba39a55564','TOOL'),(18,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/84edd0ed-f084-47bb-af11-d8b47b9f1865/call-HaplotypeCallerTask/shard-8/NexPond-445394.HC.vcf.idx','2019-03-13 00:29:21','d2fd7c70ac3bacf55cf5aec9cdee2884','TOOL'),(19,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/683dad15-3dea-4f35-8826-6d31f0e0c7bc/call-HaplotypeCallerTask/shard-0/NexPond-359781.HC.vcf','2019-03-13 00:29:38','783ee04121017664b079613654fd99c4','TOOL'),(20,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/683dad15-3dea-4f35-8826-6d31f0e0c7bc/call-HaplotypeCallerTask/shard-0/NexPond-359781.HC.vcf.idx','2019-03-13 00:29:38','4b7c3cd886bdb086725bd514c67e7081','TOOL'),(21,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/683dad15-3dea-4f35-8826-6d31f0e0c7bc/call-HaplotypeCallerTask/shard-1/NexPond-359877.HC.vcf','2019-03-13 00:29:38','fc85056661c9b3c3fb3a487844cfad90','TOOL'),(22,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/683dad15-3dea-4f35-8826-6d31f0e0c7bc/call-HaplotypeCallerTask/shard-1/NexPond-359877.HC.vcf.idx','2019-03-13 00:29:38','33ea8adb3ed152ba842e7f1f8f7c303a','TOOL'),(23,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/683dad15-3dea-4f35-8826-6d31f0e0c7bc/call-HaplotypeCallerTask/shard-2/NexPond-360361.HC.vcf','2019-03-13 00:29:38','ba9db09e845fa9fd0d14f8c12b0ff3d3','TOOL'),(24,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/683dad15-3dea-4f35-8826-6d31f0e0c7bc/call-HaplotypeCallerTask/shard-2/NexPond-360361.HC.vcf.idx','2019-03-13 00:29:38','3ff141cb604e8628202e1438be3eb07d','TOOL'),(25,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/683dad15-3dea-4f35-8826-6d31f0e0c7bc/call-HaplotypeCallerTask/shard-3/NexPond-360457.HC.vcf','2019-03-13 00:29:36','948555985c1a2e3e4fb81ca8969da006','TOOL'),(26,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/683dad15-3dea-4f35-8826-6d31f0e0c7bc/call-HaplotypeCallerTask/shard-3/NexPond-360457.HC.vcf.idx','2019-03-13 00:29:37','6514f9bb335c90a61e9d1d1491e065c4','TOOL'),(27,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/683dad15-3dea-4f35-8826-6d31f0e0c7bc/call-HaplotypeCallerTask/shard-4/NexPond-361337.HC.vcf','2019-03-13 00:29:39','65e0071f21832eb22d180c7f46016c25','TOOL'),(28,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/683dad15-3dea-4f35-8826-6d31f0e0c7bc/call-HaplotypeCallerTask/shard-4/NexPond-361337.HC.vcf.idx','2019-03-13 00:29:38','acd81313ac3f2a782616c473c2c7177f','TOOL'),(29,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/683dad15-3dea-4f35-8826-6d31f0e0c7bc/call-HaplotypeCallerTask/shard-5/NexPond-361433.HC.vcf','2019-03-13 00:29:37','801bcda564a9d77f4e9dcf3aecf69ebe','TOOL'),(30,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/683dad15-3dea-4f35-8826-6d31f0e0c7bc/call-HaplotypeCallerTask/shard-5/NexPond-361433.HC.vcf.idx','2019-03-13 00:29:41','0f1e332568a87aed712920c1639b95ab','TOOL'),(31,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/683dad15-3dea-4f35-8826-6d31f0e0c7bc/call-HaplotypeCallerTask/shard-6/NexPond-362428.HC.vcf','2019-03-13 00:29:41','ed65c327719ffd65da4a90c6d48dcf2d','TOOL'),(32,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/683dad15-3dea-4f35-8826-6d31f0e0c7bc/call-HaplotypeCallerTask/shard-6/NexPond-362428.HC.vcf.idx','2019-03-13 00:29:38','bca5d29ce0b0a0fb8f183ceb5c01eb3e','TOOL'),(33,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/683dad15-3dea-4f35-8826-6d31f0e0c7bc/call-HaplotypeCallerTask/shard-7/NexPond-363907.HC.vcf','2019-03-13 00:29:37','2b3598caa91c8ab1d0d3e6fd0a315214','TOOL'),(34,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/683dad15-3dea-4f35-8826-6d31f0e0c7bc/call-HaplotypeCallerTask/shard-7/NexPond-363907.HC.vcf.idx','2019-03-13 00:29:40','8c9bb951e2341c5b17d79727faefe6a8','TOOL'),(35,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/683dad15-3dea-4f35-8826-6d31f0e0c7bc/call-HaplotypeCallerTask/shard-8/NexPond-445394.HC.vcf','2019-03-13 00:29:38','02994f5653fa6accffd24a1c484742ce','TOOL'),(36,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/683dad15-3dea-4f35-8826-6d31f0e0c7bc/call-HaplotypeCallerTask/shard-8/NexPond-445394.HC.vcf.idx','2019-03-13 00:29:34','6354e5a99e9e6ae0f750a04d023b0f0f','TOOL'),(37,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/334ef0b9-ce97-49b3-8728-b6400396cde7/call-HaplotypeCallerTask/shard-0/NexPond-359781.HC.vcf','2019-03-13 00:29:57','698acea090eb3400cbfc25427ab097c1','TOOL'),(38,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/334ef0b9-ce97-49b3-8728-b6400396cde7/call-HaplotypeCallerTask/shard-0/NexPond-359781.HC.vcf.idx','2019-03-13 00:29:57','057f7e41cf26db5db76f4cb708b1a9e1','TOOL'),(39,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/334ef0b9-ce97-49b3-8728-b6400396cde7/call-HaplotypeCallerTask/shard-1/NexPond-359877.HC.vcf','2019-03-13 00:29:56','bfef234057956c9ede1ca4fd8864d50e','TOOL'),(40,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/334ef0b9-ce97-49b3-8728-b6400396cde7/call-HaplotypeCallerTask/shard-1/NexPond-359877.HC.vcf.idx','2019-03-13 00:29:54','1fb8130ae3f4d9b0cfdca5e5fef1616d','TOOL'),(41,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/334ef0b9-ce97-49b3-8728-b6400396cde7/call-HaplotypeCallerTask/shard-2/NexPond-360361.HC.vcf','2019-03-13 00:29:57','c2de2688281a01f58b4852adefa013aa','TOOL'),(42,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/334ef0b9-ce97-49b3-8728-b6400396cde7/call-HaplotypeCallerTask/shard-2/NexPond-360361.HC.vcf.idx','2019-03-13 00:29:57','ff36284b1d1394f7d7c63fea7368aab4','TOOL'),(43,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/334ef0b9-ce97-49b3-8728-b6400396cde7/call-HaplotypeCallerTask/shard-3/NexPond-360457.HC.vcf','2019-03-13 00:29:57','862896b1403f44bce38d877ca6a09298','TOOL'),(44,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/334ef0b9-ce97-49b3-8728-b6400396cde7/call-HaplotypeCallerTask/shard-3/NexPond-360457.HC.vcf.idx','2019-03-13 00:29:57','7af7a3df8e826641a1417300c1da29dd','TOOL'),(45,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/334ef0b9-ce97-49b3-8728-b6400396cde7/call-HaplotypeCallerTask/shard-4/NexPond-361337.HC.vcf','2019-03-13 00:29:57','d38912ff3248ddc8e5ab83b8c9242206','TOOL'),(46,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/334ef0b9-ce97-49b3-8728-b6400396cde7/call-HaplotypeCallerTask/shard-4/NexPond-361337.HC.vcf.idx','2019-03-13 00:29:56','90167cc0352575e408947ef4b50a9775','TOOL'),(47,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/334ef0b9-ce97-49b3-8728-b6400396cde7/call-HaplotypeCallerTask/shard-5/NexPond-361433.HC.vcf','2019-03-13 00:29:57','3e691cdc931e84021b938406535e3825','TOOL'),(48,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/334ef0b9-ce97-49b3-8728-b6400396cde7/call-HaplotypeCallerTask/shard-5/NexPond-361433.HC.vcf.idx','2019-03-13 00:29:56','4a24fcf02cfe82062932279cd36c370a','TOOL'),(49,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/334ef0b9-ce97-49b3-8728-b6400396cde7/call-HaplotypeCallerTask/shard-6/NexPond-362428.HC.vcf','2019-03-13 00:29:56','b6daff92a342b8c7eb00dbe58823a3c4','TOOL'),(50,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/334ef0b9-ce97-49b3-8728-b6400396cde7/call-HaplotypeCallerTask/shard-6/NexPond-362428.HC.vcf.idx','2019-03-13 00:29:57','44af87cc409b7695c8830186ac858537','TOOL'),(51,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/334ef0b9-ce97-49b3-8728-b6400396cde7/call-HaplotypeCallerTask/shard-7/NexPond-363907.HC.vcf','2019-03-13 00:29:57','7ec90ce1c73952f306f672970012073d','TOOL'),(52,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/334ef0b9-ce97-49b3-8728-b6400396cde7/call-HaplotypeCallerTask/shard-7/NexPond-363907.HC.vcf.idx','2019-03-13 00:29:57','c0503224ea883d0190311fd7c6f32ac2','TOOL'),(53,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/334ef0b9-ce97-49b3-8728-b6400396cde7/call-HaplotypeCallerTask/shard-8/NexPond-445394.HC.vcf','2019-03-13 00:29:57','3e491659ec89f1e3f2a3c24554527d4b','TOOL'),(54,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/334ef0b9-ce97-49b3-8728-b6400396cde7/call-HaplotypeCallerTask/shard-8/NexPond-445394.HC.vcf.idx','2019-03-13 00:29:56','fe25760eafaf6f2b305199bf1ff89d60','TOOL'),(55,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/be5c2094-9ed2-4898-903d-cf519128ca48/call-HaplotypeCallerTask/shard-0/NexPond-359781.HC.vcf','2019-03-13 00:30:19','85c36035e83161d8d9b628797d4d4dc1','TOOL'),(56,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/be5c2094-9ed2-4898-903d-cf519128ca48/call-HaplotypeCallerTask/shard-0/NexPond-359781.HC.vcf.idx','2019-03-13 00:30:19','f6d63f680b7013887c6188f8412a7fbc','TOOL'),(57,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/be5c2094-9ed2-4898-903d-cf519128ca48/call-HaplotypeCallerTask/shard-1/NexPond-359877.HC.vcf','2019-03-13 00:30:18','5c5a17d3f537f2107c15ced41cde3f54','TOOL'),(58,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/be5c2094-9ed2-4898-903d-cf519128ca48/call-HaplotypeCallerTask/shard-1/NexPond-359877.HC.vcf.idx','2019-03-13 00:30:19','99b005a1364325c4aa9fae4c9889d104','TOOL'),(59,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/be5c2094-9ed2-4898-903d-cf519128ca48/call-HaplotypeCallerTask/shard-2/NexPond-360361.HC.vcf','2019-03-13 00:30:18','13d3cb591f5e854f8034627dfd2645fc','TOOL'),(60,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/be5c2094-9ed2-4898-903d-cf519128ca48/call-HaplotypeCallerTask/shard-2/NexPond-360361.HC.vcf.idx','2019-03-13 00:30:19','06357ca3218a7f022a3913ed1cd46d76','TOOL'),(61,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/be5c2094-9ed2-4898-903d-cf519128ca48/call-HaplotypeCallerTask/shard-3/NexPond-360457.HC.vcf','2019-03-13 00:30:19','3c7c10ba2249e15a4cbb3f8437d98781','TOOL'),(62,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/be5c2094-9ed2-4898-903d-cf519128ca48/call-HaplotypeCallerTask/shard-3/NexPond-360457.HC.vcf.idx','2019-03-13 00:30:16','fc3993c739d88d99b9006b4ee69e28b3','TOOL'),(63,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/be5c2094-9ed2-4898-903d-cf519128ca48/call-HaplotypeCallerTask/shard-4/NexPond-361337.HC.vcf','2019-03-13 00:30:18','1b40128fd21dc5ad4e1d9c525ec7bc42','TOOL'),(64,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/be5c2094-9ed2-4898-903d-cf519128ca48/call-HaplotypeCallerTask/shard-4/NexPond-361337.HC.vcf.idx','2019-03-13 00:30:18','86b8f1f9ee27423f4325fd5f193e93b3','TOOL'),(65,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/be5c2094-9ed2-4898-903d-cf519128ca48/call-HaplotypeCallerTask/shard-5/NexPond-361433.HC.vcf','2019-03-13 00:30:18','60eddc4f95497db2893e47f4bf78b72d','TOOL'),(66,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/be5c2094-9ed2-4898-903d-cf519128ca48/call-HaplotypeCallerTask/shard-5/NexPond-361433.HC.vcf.idx','2019-03-13 00:30:19','dda7bb21f18c5565a02d5581dc2056cf','TOOL'),(67,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/be5c2094-9ed2-4898-903d-cf519128ca48/call-HaplotypeCallerTask/shard-6/NexPond-362428.HC.vcf','2019-03-13 00:30:19','595c916841c0c3dcb6c45569c1bb5ca9','TOOL'),(68,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/be5c2094-9ed2-4898-903d-cf519128ca48/call-HaplotypeCallerTask/shard-6/NexPond-362428.HC.vcf.idx','2019-03-13 00:30:19','8fae89b9682a7257f0fb7fa58a3d4990','TOOL'),(69,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/be5c2094-9ed2-4898-903d-cf519128ca48/call-HaplotypeCallerTask/shard-7/NexPond-363907.HC.vcf','2019-03-13 00:30:19','6e29ca5abbe67babaad4337457942421','TOOL'),(70,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/be5c2094-9ed2-4898-903d-cf519128ca48/call-HaplotypeCallerTask/shard-7/NexPond-363907.HC.vcf.idx','2019-03-13 00:30:19','f236eb8be7b93a50775764daa08b58af','TOOL'),(71,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/be5c2094-9ed2-4898-903d-cf519128ca48/call-HaplotypeCallerTask/shard-8/NexPond-445394.HC.vcf','2019-03-13 00:30:16','aa9ced6eee7ccc788fa8fcd06d22776f','TOOL'),(72,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/be5c2094-9ed2-4898-903d-cf519128ca48/call-HaplotypeCallerTask/shard-8/NexPond-445394.HC.vcf.idx','2019-03-13 00:30:18','b6f778c174a8b5634d5bb39853cbea1b','TOOL'),(73,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/5e7bc348-4745-4e6b-8231-bae0f57fc0b0/call-HaplotypeCallerTask/shard-0/NexPond-359781.HC.vcf','2019-03-13 00:30:39','bc3605c7456c1284e9674cca92ab6124','TOOL'),(74,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/5e7bc348-4745-4e6b-8231-bae0f57fc0b0/call-HaplotypeCallerTask/shard-0/NexPond-359781.HC.vcf.idx','2019-03-13 00:30:38','9edb7d3bf5b49264e1a8adaed11d0c91','TOOL'),(75,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/5e7bc348-4745-4e6b-8231-bae0f57fc0b0/call-HaplotypeCallerTask/shard-1/NexPond-359877.HC.vcf','2019-03-13 00:30:38','bbfc6e4ac409ac98bb2105c9bc4ce2cd','TOOL'),(76,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/5e7bc348-4745-4e6b-8231-bae0f57fc0b0/call-HaplotypeCallerTask/shard-1/NexPond-359877.HC.vcf.idx','2019-03-13 00:30:38','af2e3096e2fc3b3623ce421ce0e96b2b','TOOL'),(77,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/5e7bc348-4745-4e6b-8231-bae0f57fc0b0/call-HaplotypeCallerTask/shard-2/NexPond-360361.HC.vcf','2019-03-13 00:30:36','c6c99bef69f797dee36057a63a30522c','TOOL'),(78,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/5e7bc348-4745-4e6b-8231-bae0f57fc0b0/call-HaplotypeCallerTask/shard-2/NexPond-360361.HC.vcf.idx','2019-03-13 00:30:39','4432666f5145b6c391d0263189491a68','TOOL'),(79,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/5e7bc348-4745-4e6b-8231-bae0f57fc0b0/call-HaplotypeCallerTask/shard-3/NexPond-360457.HC.vcf','2019-03-13 00:30:38','5e4e8cbf7dfdd2dc27dfecf04c9c9fb4','TOOL'),(80,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/5e7bc348-4745-4e6b-8231-bae0f57fc0b0/call-HaplotypeCallerTask/shard-3/NexPond-360457.HC.vcf.idx','2019-03-13 00:30:39','1eea129992fd7465081adf2231c08300','TOOL'),(81,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/5e7bc348-4745-4e6b-8231-bae0f57fc0b0/call-HaplotypeCallerTask/shard-4/NexPond-361337.HC.vcf','2019-03-13 00:30:39','1ea00a41223994bb9b8ea1a73f2a0a65','TOOL'),(82,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/5e7bc348-4745-4e6b-8231-bae0f57fc0b0/call-HaplotypeCallerTask/shard-4/NexPond-361337.HC.vcf.idx','2019-03-13 00:30:39','00092d438579ad867a094ffb4a87caaf','TOOL'),(83,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/5e7bc348-4745-4e6b-8231-bae0f57fc0b0/call-HaplotypeCallerTask/shard-5/NexPond-361433.HC.vcf','2019-03-13 00:30:41','dfbd48e06d54d1d97b3161ce8557faff','TOOL'),(84,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/5e7bc348-4745-4e6b-8231-bae0f57fc0b0/call-HaplotypeCallerTask/shard-5/NexPond-361433.HC.vcf.idx','2019-03-13 00:30:38','edb3cc5b06a39d5d1b91f01d5a6eb886','TOOL'),(85,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/5e7bc348-4745-4e6b-8231-bae0f57fc0b0/call-HaplotypeCallerTask/shard-6/NexPond-362428.HC.vcf','2019-03-13 00:30:36','589400c55332d02bc2b03fefc819e6e6','TOOL'),(86,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/5e7bc348-4745-4e6b-8231-bae0f57fc0b0/call-HaplotypeCallerTask/shard-6/NexPond-362428.HC.vcf.idx','2019-03-13 00:30:39','f6218d6a8804c8e4a0d10f745b09d272','TOOL'),(87,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/5e7bc348-4745-4e6b-8231-bae0f57fc0b0/call-HaplotypeCallerTask/shard-7/NexPond-363907.HC.vcf','2019-03-13 00:30:39','90a811bc4b4e1dce82a985237c3c6c68','TOOL'),(88,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/5e7bc348-4745-4e6b-8231-bae0f57fc0b0/call-HaplotypeCallerTask/shard-7/NexPond-363907.HC.vcf.idx','2019-03-13 00:30:38','d73edfffc7c46da887116d587b8adf2f','TOOL'),(89,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/5e7bc348-4745-4e6b-8231-bae0f57fc0b0/call-HaplotypeCallerTask/shard-8/NexPond-445394.HC.vcf','2019-03-13 00:30:38','aa03df230b5c0cda4cee6012def8f72b','TOOL'),(90,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/5e7bc348-4745-4e6b-8231-bae0f57fc0b0/call-HaplotypeCallerTask/shard-8/NexPond-445394.HC.vcf.idx','2019-03-13 00:30:39','e0154253e841b58dcfbf6deb0d3438f4','TOOL'),(91,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/73a9ee75-6006-40f4-8f1b-681c38a501a8/call-HaplotypeCallerTask/shard-0/NexPond-359781.HC.vcf','2019-03-13 00:30:58','dee2f453ed4945620c4776c99ec92ee3','TOOL'),(92,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/73a9ee75-6006-40f4-8f1b-681c38a501a8/call-HaplotypeCallerTask/shard-0/NexPond-359781.HC.vcf.idx','2019-03-13 00:30:59','286c27d1c195349266e1b966c8da1e8b','TOOL'),(93,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/73a9ee75-6006-40f4-8f1b-681c38a501a8/call-HaplotypeCallerTask/shard-1/NexPond-359877.HC.vcf','2019-03-13 00:30:59','ec90827c325b3c47138a6a52974c26be','TOOL'),(94,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/73a9ee75-6006-40f4-8f1b-681c38a501a8/call-HaplotypeCallerTask/shard-1/NexPond-359877.HC.vcf.idx','2019-03-13 00:30:58','309006f418e12415af19ae5e1e830414','TOOL'),(95,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/73a9ee75-6006-40f4-8f1b-681c38a501a8/call-HaplotypeCallerTask/shard-2/NexPond-360361.HC.vcf','2019-03-13 00:30:58','65e4b9c96e1b0ef515bca0b8a1fc6d94','TOOL'),(96,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/73a9ee75-6006-40f4-8f1b-681c38a501a8/call-HaplotypeCallerTask/shard-2/NexPond-360361.HC.vcf.idx','2019-03-13 00:30:58','59e1164c9f75622b11a276b4ef541feb','TOOL'),(97,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/73a9ee75-6006-40f4-8f1b-681c38a501a8/call-HaplotypeCallerTask/shard-3/NexPond-360457.HC.vcf','2019-03-13 00:30:58','ea20f3bf8a2cadf84a4c430436128d89','TOOL'),(98,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/73a9ee75-6006-40f4-8f1b-681c38a501a8/call-HaplotypeCallerTask/shard-3/NexPond-360457.HC.vcf.idx','2019-03-13 00:30:59','ab7400662d1e16af36da71f3e618b3e1','TOOL'),(99,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/73a9ee75-6006-40f4-8f1b-681c38a501a8/call-HaplotypeCallerTask/shard-4/NexPond-361337.HC.vcf','2019-03-13 00:30:58','d827f144ebc2d5a4a1865da6fce61a1f','TOOL'),(100,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/73a9ee75-6006-40f4-8f1b-681c38a501a8/call-HaplotypeCallerTask/shard-4/NexPond-361337.HC.vcf.idx','2019-03-13 00:30:58','0baa8bd9a8a1c6e553b8084ac40f3c7f','TOOL'),(101,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/73a9ee75-6006-40f4-8f1b-681c38a501a8/call-HaplotypeCallerTask/shard-5/NexPond-361433.HC.vcf','2019-03-13 00:30:58','a1d6aff44d3976963cd64dd32537d87b','TOOL'),(102,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/73a9ee75-6006-40f4-8f1b-681c38a501a8/call-HaplotypeCallerTask/shard-5/NexPond-361433.HC.vcf.idx','2019-03-13 00:30:59','311dac32853d12060a3fde1073d1ac0b','TOOL'),(103,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/73a9ee75-6006-40f4-8f1b-681c38a501a8/call-HaplotypeCallerTask/shard-6/NexPond-362428.HC.vcf','2019-03-13 00:30:58','c5d7ded290c766399e47fa3836a49cc5','TOOL'),(104,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/73a9ee75-6006-40f4-8f1b-681c38a501a8/call-HaplotypeCallerTask/shard-6/NexPond-362428.HC.vcf.idx','2019-03-13 00:30:59','0ee424d8985bbcb0ee730f86f521b7f5','TOOL'),(105,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/73a9ee75-6006-40f4-8f1b-681c38a501a8/call-HaplotypeCallerTask/shard-7/NexPond-363907.HC.vcf','2019-03-13 00:30:59','d88075a5bfaa3ddf61610c0e36bf07db','TOOL'),(106,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/73a9ee75-6006-40f4-8f1b-681c38a501a8/call-HaplotypeCallerTask/shard-7/NexPond-363907.HC.vcf.idx','2019-03-13 00:30:58','39f4b3c37aa6ccc55d15536fc9cede0f','TOOL'),(107,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/73a9ee75-6006-40f4-8f1b-681c38a501a8/call-HaplotypeCallerTask/shard-8/NexPond-445394.HC.vcf','2019-03-13 00:30:58','8cee4d226597d69167df36adea7fbeca','TOOL'),(108,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/73a9ee75-6006-40f4-8f1b-681c38a501a8/call-HaplotypeCallerTask/shard-8/NexPond-445394.HC.vcf.idx','2019-03-13 00:30:58','e5216279b00f3847623143fdf17f3bd4','TOOL'),(109,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/662f5bfb-038c-491d-925b-896cc1038ff2/call-HaplotypeCallerTask/shard-0/NexPond-359781.HC.vcf','2019-03-13 00:31:19','3cd00d948445485e22bfac3e6abdd422','TOOL'),(110,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/662f5bfb-038c-491d-925b-896cc1038ff2/call-HaplotypeCallerTask/shard-0/NexPond-359781.HC.vcf.idx','2019-03-13 00:31:19','2cc033d98826fe406431735beb9a73b9','TOOL'),(111,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/662f5bfb-038c-491d-925b-896cc1038ff2/call-HaplotypeCallerTask/shard-1/NexPond-359877.HC.vcf','2019-03-13 00:31:18','db9ec284dc0341fc9816960084991742','TOOL'),(112,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/662f5bfb-038c-491d-925b-896cc1038ff2/call-HaplotypeCallerTask/shard-1/NexPond-359877.HC.vcf.idx','2019-03-13 00:31:18','1363b31d76febb3832d78406b189b864','TOOL'),(113,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/662f5bfb-038c-491d-925b-896cc1038ff2/call-HaplotypeCallerTask/shard-2/NexPond-360361.HC.vcf','2019-03-13 00:31:21','3556f9769d904abbcc7f1592fe3a23ef','TOOL'),(114,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/662f5bfb-038c-491d-925b-896cc1038ff2/call-HaplotypeCallerTask/shard-2/NexPond-360361.HC.vcf.idx','2019-03-13 00:31:21','558cc5e193ecfa3d135900c168a51be9','TOOL'),(115,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/662f5bfb-038c-491d-925b-896cc1038ff2/call-HaplotypeCallerTask/shard-3/NexPond-360457.HC.vcf','2019-03-13 00:31:19','d957ecb60f0bbafb44ba5f58dda16ee0','TOOL'),(116,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/662f5bfb-038c-491d-925b-896cc1038ff2/call-HaplotypeCallerTask/shard-3/NexPond-360457.HC.vcf.idx','2019-03-13 00:31:18','e9ca65e37f14e5c06aa57af9813285ce','TOOL'),(117,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/662f5bfb-038c-491d-925b-896cc1038ff2/call-HaplotypeCallerTask/shard-4/NexPond-361337.HC.vcf','2019-03-13 00:31:18','daf32ab4ab8673422bd557595a55d113','TOOL'),(118,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/662f5bfb-038c-491d-925b-896cc1038ff2/call-HaplotypeCallerTask/shard-4/NexPond-361337.HC.vcf.idx','2019-03-13 00:31:19','4bf8d9f2ecdcdd4e0d04199a5f5eba21','TOOL'),(119,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/662f5bfb-038c-491d-925b-896cc1038ff2/call-HaplotypeCallerTask/shard-5/NexPond-361433.HC.vcf','2019-03-13 00:31:19','e4ae9b240ce3afffee31ea0dbcf9fd60','TOOL'),(120,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/662f5bfb-038c-491d-925b-896cc1038ff2/call-HaplotypeCallerTask/shard-5/NexPond-361433.HC.vcf.idx','2019-03-13 00:31:18','1f37c1c580b044025a697d0a2e9616e8','TOOL'),(121,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/662f5bfb-038c-491d-925b-896cc1038ff2/call-HaplotypeCallerTask/shard-6/NexPond-362428.HC.vcf','2019-03-13 00:31:18','b9cdfd9c270a4e70930fdf69e2aa1fab','TOOL'),(122,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/662f5bfb-038c-491d-925b-896cc1038ff2/call-HaplotypeCallerTask/shard-6/NexPond-362428.HC.vcf.idx','2019-03-13 00:31:18','98931343e5c08f50a183218aa77dcb30','TOOL'),(123,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/662f5bfb-038c-491d-925b-896cc1038ff2/call-HaplotypeCallerTask/shard-7/NexPond-363907.HC.vcf','2019-03-13 00:31:16','2aa67774456100045615172c6839c2c7','TOOL'),(124,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/662f5bfb-038c-491d-925b-896cc1038ff2/call-HaplotypeCallerTask/shard-7/NexPond-363907.HC.vcf.idx','2019-03-13 00:31:18','0779ba8efccec79b6fea5955e7e7a2b6','TOOL'),(125,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/662f5bfb-038c-491d-925b-896cc1038ff2/call-HaplotypeCallerTask/shard-8/NexPond-445394.HC.vcf','2019-03-13 00:31:18','431b0be5d711920132091abb46862a14','TOOL'),(126,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/662f5bfb-038c-491d-925b-896cc1038ff2/call-HaplotypeCallerTask/shard-8/NexPond-445394.HC.vcf.idx','2019-03-13 00:31:18','7cc4127f8428bc497171f661f89ee7bd','TOOL'),(127,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/301fbc8e-2be1-41bd-847f-0dc4aff9f9af/call-HaplotypeCallerTask/shard-0/NexPond-359781.HC.vcf','2019-03-13 00:31:38','efc06df3d1679aac02fbebfd207d0cf8','TOOL'),(128,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/301fbc8e-2be1-41bd-847f-0dc4aff9f9af/call-HaplotypeCallerTask/shard-0/NexPond-359781.HC.vcf.idx','2019-03-13 00:31:38','aeeecd201df086a79d65b6e8c3a14a9c','TOOL'),(129,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/301fbc8e-2be1-41bd-847f-0dc4aff9f9af/call-HaplotypeCallerTask/shard-1/NexPond-359877.HC.vcf','2019-03-13 00:31:38','3920e7c43dfc40d70abcd4bd62e26eff','TOOL'),(130,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/301fbc8e-2be1-41bd-847f-0dc4aff9f9af/call-HaplotypeCallerTask/shard-1/NexPond-359877.HC.vcf.idx','2019-03-13 00:31:41','ccef09df22737a85d850b29128ab1ef8','TOOL'),(131,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/301fbc8e-2be1-41bd-847f-0dc4aff9f9af/call-HaplotypeCallerTask/shard-2/NexPond-360361.HC.vcf','2019-03-13 00:31:38','cb87b8f8c20254a80506a958a2964d80','TOOL'),(132,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/301fbc8e-2be1-41bd-847f-0dc4aff9f9af/call-HaplotypeCallerTask/shard-2/NexPond-360361.HC.vcf.idx','2019-03-13 00:31:41','dd35e90afb39b43e9cace8b52f433528','TOOL'),(133,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/301fbc8e-2be1-41bd-847f-0dc4aff9f9af/call-HaplotypeCallerTask/shard-3/NexPond-360457.HC.vcf','2019-03-13 00:31:41','dfec481cdcd9de9c8d38cafc32e39f98','TOOL'),(134,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/301fbc8e-2be1-41bd-847f-0dc4aff9f9af/call-HaplotypeCallerTask/shard-3/NexPond-360457.HC.vcf.idx','2019-03-13 00:31:38','549eb124d5d3da46e75ae0d65e12c922','TOOL'),(135,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/301fbc8e-2be1-41bd-847f-0dc4aff9f9af/call-HaplotypeCallerTask/shard-4/NexPond-361337.HC.vcf','2019-03-13 00:31:38','e6c5d5de71a34a8df834ca79215715d1','TOOL'),(136,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/301fbc8e-2be1-41bd-847f-0dc4aff9f9af/call-HaplotypeCallerTask/shard-4/NexPond-361337.HC.vcf.idx','2019-03-13 00:31:38','e4cb0f0f2fd561978c896134cfff3b07','TOOL'),(137,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/301fbc8e-2be1-41bd-847f-0dc4aff9f9af/call-HaplotypeCallerTask/shard-5/NexPond-361433.HC.vcf','2019-03-13 00:31:38','64e57ff5ef452a68c68255ecad11430a','TOOL'),(138,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/301fbc8e-2be1-41bd-847f-0dc4aff9f9af/call-HaplotypeCallerTask/shard-5/NexPond-361433.HC.vcf.idx','2019-03-13 00:31:39','98a7bb18411cc729550caf505b22fab5','TOOL'),(139,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/301fbc8e-2be1-41bd-847f-0dc4aff9f9af/call-HaplotypeCallerTask/shard-6/NexPond-362428.HC.vcf','2019-03-13 00:31:39','8c4cacf3c3f37c098029f0a8af3d887d','TOOL'),(140,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/301fbc8e-2be1-41bd-847f-0dc4aff9f9af/call-HaplotypeCallerTask/shard-6/NexPond-362428.HC.vcf.idx','2019-03-13 00:31:41','17b031002157f3316070c6255a9e5661','TOOL'),(141,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/301fbc8e-2be1-41bd-847f-0dc4aff9f9af/call-HaplotypeCallerTask/shard-7/NexPond-363907.HC.vcf','2019-03-13 00:31:41','63e2571b55b3cc44aaf0fa843e5cebfc','TOOL'),(142,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/301fbc8e-2be1-41bd-847f-0dc4aff9f9af/call-HaplotypeCallerTask/shard-7/NexPond-363907.HC.vcf.idx','2019-03-13 00:31:38','6342b1203c02a8f6d6864604a9fcff90','TOOL'),(143,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/301fbc8e-2be1-41bd-847f-0dc4aff9f9af/call-HaplotypeCallerTask/shard-8/NexPond-445394.HC.vcf','2019-03-13 00:31:39','02763c9ff14dc4658af1a4f858ec3507','TOOL'),(144,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/301fbc8e-2be1-41bd-847f-0dc4aff9f9af/call-HaplotypeCallerTask/shard-8/NexPond-445394.HC.vcf.idx','2019-03-13 00:31:38','d5a3ea5ba63028465676e07826e72131','TOOL'),(145,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/001d8293-6c8c-41b9-8f07-274e49039d9b/call-HaplotypeCallerTask/shard-0/NexPond-359781.HC.vcf','2019-03-13 00:31:58','42be51c1f0c5f14d396f9114b448edb8','TOOL'),(146,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/001d8293-6c8c-41b9-8f07-274e49039d9b/call-HaplotypeCallerTask/shard-0/NexPond-359781.HC.vcf.idx','2019-03-13 00:32:01','93d5ecc09ce81963e87743f1e55c1999','TOOL'),(147,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/001d8293-6c8c-41b9-8f07-274e49039d9b/call-HaplotypeCallerTask/shard-1/NexPond-359877.HC.vcf','2019-03-13 00:32:01','148ae3ed723f8609b37e35b8968cb565','TOOL'),(148,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/001d8293-6c8c-41b9-8f07-274e49039d9b/call-HaplotypeCallerTask/shard-1/NexPond-359877.HC.vcf.idx','2019-03-13 00:31:58','cac296fb7d41ce440b2f4224cd0d7d58','TOOL'),(149,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/001d8293-6c8c-41b9-8f07-274e49039d9b/call-HaplotypeCallerTask/shard-2/NexPond-360361.HC.vcf','2019-03-13 00:31:58','e6a6cef2462360d433e024fb7750fb07','TOOL'),(150,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/001d8293-6c8c-41b9-8f07-274e49039d9b/call-HaplotypeCallerTask/shard-2/NexPond-360361.HC.vcf.idx','2019-03-13 00:31:59','ef2e497cf170f8d70e70621144dfee86','TOOL'),(151,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/001d8293-6c8c-41b9-8f07-274e49039d9b/call-HaplotypeCallerTask/shard-3/NexPond-360457.HC.vcf','2019-03-13 00:31:59','65ab1cf323edc3f796a51589f2de0d37','TOOL'),(152,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/001d8293-6c8c-41b9-8f07-274e49039d9b/call-HaplotypeCallerTask/shard-3/NexPond-360457.HC.vcf.idx','2019-03-13 00:31:58','3373e4264c63734418c84f2ef0170a4e','TOOL'),(153,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/001d8293-6c8c-41b9-8f07-274e49039d9b/call-HaplotypeCallerTask/shard-4/NexPond-361337.HC.vcf','2019-03-13 00:31:59','529daeb5087c44c0bf483b275dfcac51','TOOL'),(154,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/001d8293-6c8c-41b9-8f07-274e49039d9b/call-HaplotypeCallerTask/shard-4/NexPond-361337.HC.vcf.idx','2019-03-13 00:32:01','3ab0d1d23df0004d3e25f6745ca04b60','TOOL'),(155,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/001d8293-6c8c-41b9-8f07-274e49039d9b/call-HaplotypeCallerTask/shard-5/NexPond-361433.HC.vcf','2019-03-13 00:31:59','231f00b996ad70c079097073b34de087','TOOL'),(156,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/001d8293-6c8c-41b9-8f07-274e49039d9b/call-HaplotypeCallerTask/shard-5/NexPond-361433.HC.vcf.idx','2019-03-13 00:31:59','57a3692a8e90a22fe7a56cf1ee841068','TOOL'),(157,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/001d8293-6c8c-41b9-8f07-274e49039d9b/call-HaplotypeCallerTask/shard-6/NexPond-362428.HC.vcf','2019-03-13 00:31:58','756fb8fb6516673c76eb06c5e13ae374','TOOL'),(158,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/001d8293-6c8c-41b9-8f07-274e49039d9b/call-HaplotypeCallerTask/shard-6/NexPond-362428.HC.vcf.idx','2019-03-13 00:32:01','d6e9d148acec79a9f563764000bf8393','TOOL'),(159,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/001d8293-6c8c-41b9-8f07-274e49039d9b/call-HaplotypeCallerTask/shard-7/NexPond-363907.HC.vcf','2019-03-13 00:32:01','00d4b7ab63a9cbedd746ec964df7b8d0','TOOL'),(160,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/001d8293-6c8c-41b9-8f07-274e49039d9b/call-HaplotypeCallerTask/shard-7/NexPond-363907.HC.vcf.idx','2019-03-13 00:31:58','47762f0289c894286cce8046f387f0cd','TOOL'),(161,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/001d8293-6c8c-41b9-8f07-274e49039d9b/call-HaplotypeCallerTask/shard-8/NexPond-445394.HC.vcf','2019-03-13 00:31:58','21506dcc74db097248d9adb435612b72','TOOL'),(162,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/001d8293-6c8c-41b9-8f07-274e49039d9b/call-HaplotypeCallerTask/shard-8/NexPond-445394.HC.vcf.idx','2019-03-13 00:32:01','cea055c5fc86ecd0fb381c0cf17b4714','TOOL'),(163,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/1b557c70-a9e8-40f7-a4b2-d33c792c0255/call-HaplotypeCallerTask/shard-0/NexPond-359781.HC.vcf','2019-03-13 00:32:41','be4fb6950c103d158b375a7cecf51bdb','TOOL'),(164,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/1b557c70-a9e8-40f7-a4b2-d33c792c0255/call-HaplotypeCallerTask/shard-0/NexPond-359781.HC.vcf.idx','2019-03-13 00:32:38','25f7225c14b45b08b8b485bd0372e04e','TOOL'),(165,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/1b557c70-a9e8-40f7-a4b2-d33c792c0255/call-HaplotypeCallerTask/shard-1/NexPond-359877.HC.vcf','2019-03-13 00:32:38','ae02293aa2a344223a430b638a0dfba8','TOOL'),(166,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/1b557c70-a9e8-40f7-a4b2-d33c792c0255/call-HaplotypeCallerTask/shard-1/NexPond-359877.HC.vcf.idx','2019-03-13 00:32:38','0db3def9525adf543cac15e47bebbb9c','TOOL'),(167,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/1b557c70-a9e8-40f7-a4b2-d33c792c0255/call-HaplotypeCallerTask/shard-2/NexPond-360361.HC.vcf','2019-03-13 00:32:39','42ad0d0aaeb3e19893d430b9db3c6ca7','TOOL'),(168,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/1b557c70-a9e8-40f7-a4b2-d33c792c0255/call-HaplotypeCallerTask/shard-2/NexPond-360361.HC.vcf.idx','2019-03-13 00:32:39','378da441dc643adc16ad0ea48637b295','TOOL'),(169,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/1b557c70-a9e8-40f7-a4b2-d33c792c0255/call-HaplotypeCallerTask/shard-3/NexPond-360457.HC.vcf','2019-03-13 00:32:39','ce16c6d34b8dba96e6129eadbd0dcca6','TOOL'),(170,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/1b557c70-a9e8-40f7-a4b2-d33c792c0255/call-HaplotypeCallerTask/shard-3/NexPond-360457.HC.vcf.idx','2019-03-13 00:32:39','0ba96730104f416d8636f04cfeee7e99','TOOL'),(171,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/1b557c70-a9e8-40f7-a4b2-d33c792c0255/call-HaplotypeCallerTask/shard-4/NexPond-361337.HC.vcf','2019-03-13 00:32:38','b236d400c85f025f23e798e898fc22ca','TOOL'),(172,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/1b557c70-a9e8-40f7-a4b2-d33c792c0255/call-HaplotypeCallerTask/shard-4/NexPond-361337.HC.vcf.idx','2019-03-13 00:32:41','14c1bdc89de9ef0b2074509278a4933e','TOOL'),(173,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/1b557c70-a9e8-40f7-a4b2-d33c792c0255/call-HaplotypeCallerTask/shard-5/NexPond-361433.HC.vcf','2019-03-13 00:32:39','654d00862253bdf20b29348d6cf7c6d6','TOOL'),(174,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/1b557c70-a9e8-40f7-a4b2-d33c792c0255/call-HaplotypeCallerTask/shard-5/NexPond-361433.HC.vcf.idx','2019-03-13 00:32:39','0f4bbe0e46a2439a5bf1a51d7491ec83','TOOL'),(175,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/1b557c70-a9e8-40f7-a4b2-d33c792c0255/call-HaplotypeCallerTask/shard-6/NexPond-362428.HC.vcf','2019-03-13 00:32:41','8a9d7ef5516ff8ae0bf39ffde4391c5b','TOOL'),(176,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/1b557c70-a9e8-40f7-a4b2-d33c792c0255/call-HaplotypeCallerTask/shard-6/NexPond-362428.HC.vcf.idx','2019-03-13 00:32:38','98d53dee97537b1e614a103a6a5f42fd','TOOL'),(177,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/1b557c70-a9e8-40f7-a4b2-d33c792c0255/call-HaplotypeCallerTask/shard-7/NexPond-363907.HC.vcf','2019-03-13 00:32:41','70bc001faf136c75620e143410e01c65','TOOL'),(178,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/1b557c70-a9e8-40f7-a4b2-d33c792c0255/call-HaplotypeCallerTask/shard-7/NexPond-363907.HC.vcf.idx','2019-03-13 00:32:39','39437b1b3dd08fc5d45419db1bc772cb','TOOL'),(179,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/1b557c70-a9e8-40f7-a4b2-d33c792c0255/call-HaplotypeCallerTask/shard-8/NexPond-445394.HC.vcf','2019-03-13 00:32:38','1913b51058ed0cb99f1b1240159f6933','TOOL'),(180,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/1b557c70-a9e8-40f7-a4b2-d33c792c0255/call-HaplotypeCallerTask/shard-8/NexPond-445394.HC.vcf.idx','2019-03-13 00:32:38','2772ce4075901bc381f4a2312d48158c','TOOL'),(181,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/88aba234-c27b-4bad-842e-9662704d64ca/call-HaplotypeCallerTask/shard-0/NexPond-359781.HC.vcf','2019-03-13 00:33:01','3953567d35a7b7437a9b897e8620c615','TOOL'),(182,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/88aba234-c27b-4bad-842e-9662704d64ca/call-HaplotypeCallerTask/shard-0/NexPond-359781.HC.vcf.idx','2019-03-13 00:32:58','87db0c35d98932841f72ad5e3cb165df','TOOL'),(183,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/88aba234-c27b-4bad-842e-9662704d64ca/call-HaplotypeCallerTask/shard-1/NexPond-359877.HC.vcf','2019-03-13 00:32:58','bb26da06ddc81192a64f589fa8e1ed00','TOOL'),(184,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/88aba234-c27b-4bad-842e-9662704d64ca/call-HaplotypeCallerTask/shard-1/NexPond-359877.HC.vcf.idx','2019-03-13 00:32:59','40cd93b8e3f51bbfcde766df45a2c1a5','TOOL'),(185,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/88aba234-c27b-4bad-842e-9662704d64ca/call-HaplotypeCallerTask/shard-2/NexPond-360361.HC.vcf','2019-03-13 00:32:58','5821727f33a585e551a0453d20701803','TOOL'),(186,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/88aba234-c27b-4bad-842e-9662704d64ca/call-HaplotypeCallerTask/shard-2/NexPond-360361.HC.vcf.idx','2019-03-13 00:32:59','c02d72d78d0ea434703180782cf6c25e','TOOL'),(187,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/88aba234-c27b-4bad-842e-9662704d64ca/call-HaplotypeCallerTask/shard-3/NexPond-360457.HC.vcf','2019-03-13 00:33:01','92d753c65222759fe3a6ff593bbc95eb','TOOL'),(188,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/88aba234-c27b-4bad-842e-9662704d64ca/call-HaplotypeCallerTask/shard-3/NexPond-360457.HC.vcf.idx','2019-03-13 00:32:58','822b39cdd319aef3186371fbb4f3f8fe','TOOL'),(189,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/88aba234-c27b-4bad-842e-9662704d64ca/call-HaplotypeCallerTask/shard-4/NexPond-361337.HC.vcf','2019-03-13 00:32:58','120f59f7335e26770c5dbeb3a3a25a45','TOOL'),(190,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/88aba234-c27b-4bad-842e-9662704d64ca/call-HaplotypeCallerTask/shard-4/NexPond-361337.HC.vcf.idx','2019-03-13 00:33:01','bc4ddab88d13432524dae3b110f32559','TOOL'),(191,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/88aba234-c27b-4bad-842e-9662704d64ca/call-HaplotypeCallerTask/shard-5/NexPond-361433.HC.vcf','2019-03-13 00:32:58','3ca41fe3f6390a59169098ac36a15e46','TOOL'),(192,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/88aba234-c27b-4bad-842e-9662704d64ca/call-HaplotypeCallerTask/shard-5/NexPond-361433.HC.vcf.idx','2019-03-13 00:32:59','d71a3a332a1174a6b2eacb960e7ae6d5','TOOL'),(193,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/88aba234-c27b-4bad-842e-9662704d64ca/call-HaplotypeCallerTask/shard-6/NexPond-362428.HC.vcf','2019-03-13 00:32:59','43bb8d04d1d3a66d9ecdbe0c8c66391c','TOOL'),(194,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/88aba234-c27b-4bad-842e-9662704d64ca/call-HaplotypeCallerTask/shard-6/NexPond-362428.HC.vcf.idx','2019-03-13 00:32:58','402e6c0b82dc596e30b0de11a00a328d','TOOL'),(195,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/88aba234-c27b-4bad-842e-9662704d64ca/call-HaplotypeCallerTask/shard-7/NexPond-363907.HC.vcf','2019-03-13 00:32:59','826f0ed26eb9aa4e65f39944f04602d0','TOOL'),(196,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/88aba234-c27b-4bad-842e-9662704d64ca/call-HaplotypeCallerTask/shard-7/NexPond-363907.HC.vcf.idx','2019-03-13 00:32:58','112128f2ac71daf9c8e5a832fd76353d','TOOL'),(197,'SOMATIC_VCF','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/88aba234-c27b-4bad-842e-9662704d64ca/call-HaplotypeCallerTask/shard-8/NexPond-445394.HC.vcf','2019-03-13 00:32:58','2a50b0447efb6c6859b60a5b0aaf3af1','TOOL'),(198,'SOMATIC_VCF_INDEX','gs://broad-dsde-methods/cromwell-execution-36/ToolComparisonWdl/88aba234-c27b-4bad-842e-9662704d64ca/call-HaplotypeCallerTask/shard-8/NexPond-445394.HC.vcf.idx','2019-03-13 00:32:59','ce9ffc35e9cde9957853db8a3193df77','TOOL');
/*!40000 ALTER TABLE `OutputFiles` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `TestReport`
--

DROP TABLE IF EXISTS `TestReport`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `TestReport` (
  `idTestReport` int(11) unsigned NOT NULL AUTO_INCREMENT,
  PRIMARY KEY (`idTestReport`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `TestReport`
--

LOCK TABLES `TestReport` WRITE;
/*!40000 ALTER TABLE `TestReport` DISABLE KEYS */;
/*!40000 ALTER TABLE `TestReport` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `TestReportData`
--

DROP TABLE IF EXISTS `TestReportData`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `TestReportData` (
  `reportID` int(11) unsigned NOT NULL,
  `dataID` int(11) unsigned NOT NULL,
  PRIMARY KEY (`reportID`,`dataID`),
  KEY `TestReportData_dataID_idx` (`dataID`),
  CONSTRAINT `TestReportData_dataID` FOREIGN KEY (`dataID`) REFERENCES `OutputFiles` (`idOutputFiles`) ON DELETE NO ACTION ON UPDATE NO ACTION,
  CONSTRAINT `TestReportData_reportID` FOREIGN KEY (`reportID`) REFERENCES `TestReport` (`idTestReport`) ON DELETE NO ACTION ON UPDATE NO ACTION
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `TestReportData`
--

LOCK TABLES `TestReportData` WRITE;
/*!40000 ALTER TABLE `TestReportData` DISABLE KEYS */;
/*!40000 ALTER TABLE `TestReportData` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `TestReportMetrics`
--

DROP TABLE IF EXISTS `TestReportMetrics`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `TestReportMetrics` (
  `reportId` int(11) unsigned NOT NULL,
  `metricId` int(11) unsigned NOT NULL,
  KEY `TestReportMetrics_reportID_idx` (`reportId`),
  KEY `TestReportMetrics_metricID_idx` (`metricId`),
  CONSTRAINT `TestReportMetrics_metricID` FOREIGN KEY (`metricId`) REFERENCES `Metrics` (`idMetrics`) ON DELETE NO ACTION ON UPDATE NO ACTION,
  CONSTRAINT `TestReportMetrics_reportID` FOREIGN KEY (`reportId`) REFERENCES `TestReport` (`idTestReport`) ON DELETE NO ACTION ON UPDATE NO ACTION
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `TestReportMetrics`
--

LOCK TABLES `TestReportMetrics` WRITE;
/*!40000 ALTER TABLE `TestReportMetrics` DISABLE KEYS */;
/*!40000 ALTER TABLE `TestReportMetrics` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `TestScenario`
--

DROP TABLE IF EXISTS `TestScenario`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `TestScenario` (
  `idTestScenarioRun` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `scenarioInfo` int(11) unsigned NOT NULL,
  `configuration` json NOT NULL,
  `inputFile` int(11) unsigned NOT NULL,
  `tool` int(11) unsigned NOT NULL,
  PRIMARY KEY (`idTestScenarioRun`),
  UNIQUE KEY `idTestScenario_UNIQUE` (`idTestScenarioRun`),
  KEY `TestScenario_fileID_idx` (`inputFile`),
  KEY `TestScenario_toolID_idx` (`tool`),
  KEY `TestScenario_scenarioInfoID_idx` (`scenarioInfo`),
  CONSTRAINT `TestScenario_fileID` FOREIGN KEY (`inputFile`) REFERENCES `BaselineInputFiles` (`idBaselineInputFiles`) ON DELETE NO ACTION ON UPDATE NO ACTION,
  CONSTRAINT `TestScenario_scenarioInfoID` FOREIGN KEY (`scenarioInfo`) REFERENCES `TestScenarioInfo` (`idTestScenarioInfo`) ON DELETE NO ACTION ON UPDATE NO ACTION,
  CONSTRAINT `TestScenario_toolID` FOREIGN KEY (`tool`) REFERENCES `Tools` (`idTools`) ON DELETE NO ACTION ON UPDATE NO ACTION
) ENGINE=InnoDB AUTO_INCREMENT=100 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `TestScenario`
--

LOCK TABLES `TestScenario` WRITE;
/*!40000 ALTER TABLE `TestScenario` DISABLE KEYS */;
INSERT INTO `TestScenario` VALUES (1,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-359781.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-07-4.1.0.0-50-g342569ac0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',2,1),(2,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-359877.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-07-4.1.0.0-50-g342569ac0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',6,1),(3,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-360361.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-07-4.1.0.0-50-g342569ac0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',10,1),(4,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-360457.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-07-4.1.0.0-50-g342569ac0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',14,1),(5,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-361337.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-07-4.1.0.0-50-g342569ac0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',18,1),(6,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-361433.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-07-4.1.0.0-50-g342569ac0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',22,1),(7,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-362428.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-07-4.1.0.0-50-g342569ac0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',26,1),(8,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-363907.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-07-4.1.0.0-50-g342569ac0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',30,1),(9,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-445394.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-07-4.1.0.0-50-g342569ac0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',34,1),(10,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-359781.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.4.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',2,2),(11,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-359877.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.4.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',6,2),(12,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-360361.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.4.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',10,2),(13,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-360457.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.4.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',14,2),(14,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-361337.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.4.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',18,2),(15,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-361433.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.4.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',22,2),(16,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-362428.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.4.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',26,2),(17,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-363907.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.4.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',30,2),(18,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-445394.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.4.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',34,2),(19,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-359781.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.6.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',2,3),(20,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-359877.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.6.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',6,3),(21,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-360361.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.6.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',10,3),(22,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-360457.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.6.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',14,3),(23,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-361337.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.6.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',18,3),(24,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-361433.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.6.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',22,3),(25,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-362428.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.6.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',26,3),(26,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-363907.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.6.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',30,3),(27,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-445394.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.6.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',34,3),(28,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-359781.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.7.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',2,4),(29,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-359877.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.7.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',6,4),(30,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-360361.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.7.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',10,4),(31,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-360457.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.7.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',14,4),(32,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-361337.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.7.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',18,4),(33,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-361433.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.7.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',22,4),(34,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-362428.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.7.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',26,4),(35,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-363907.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.7.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',30,4),(36,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-445394.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.7.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',34,4),(37,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-359781.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.8.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',2,5),(38,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-359877.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.8.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',6,5),(39,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-360361.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.8.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',10,5),(40,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-360457.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.8.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',14,5),(41,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-361337.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.8.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',18,5),(42,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-361433.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.8.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',22,5),(43,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-362428.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.8.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',26,5),(44,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-363907.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.8.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',30,5),(45,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-445394.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.8.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',34,5),(46,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-359781.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.8.1-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',2,6),(47,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-359877.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.8.1-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',6,6),(48,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-360361.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.8.1-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',10,6),(49,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-360457.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.8.1-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',14,6),(50,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-361337.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.8.1-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',18,6),(51,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-361433.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.8.1-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',22,6),(52,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-362428.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.8.1-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',26,6),(53,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-363907.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.8.1-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',30,6),(54,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-445394.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.8.1-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',34,6),(55,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-359781.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.9.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',2,7),(56,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-359877.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.9.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',6,7),(57,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-360361.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.9.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',10,7),(58,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-360457.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.9.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',14,7),(59,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-361337.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.9.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',18,7),(60,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-361433.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.9.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',22,7),(61,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-362428.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.9.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',26,7),(62,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-363907.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.9.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',30,7),(63,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-445394.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.9.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',34,7),(64,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-359781.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.10.1-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',2,8),(65,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-359877.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.10.1-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',6,8),(66,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-360361.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.10.1-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',10,8),(67,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-360457.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.10.1-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',14,8),(68,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-361337.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.10.1-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',18,8),(69,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-361433.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.10.1-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',22,8),(70,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-362428.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.10.1-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',26,8),(71,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-363907.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.10.1-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',30,8),(72,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-445394.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.10.1-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',34,8),(73,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-359781.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.10.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',2,9),(74,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-359877.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.10.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',6,9),(75,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-360361.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.10.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',10,9),(76,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-360457.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.10.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',14,9),(77,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-361337.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.10.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',18,9),(78,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-361433.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.10.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',22,9),(79,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-362428.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.10.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',26,9),(80,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-363907.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.10.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',30,9),(81,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-445394.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.10.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',34,9),(82,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-359781.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.12.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',2,10),(83,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-359877.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.12.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',6,10),(84,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-360361.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.12.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',10,10),(85,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-360457.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.12.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',14,10),(86,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-361337.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.12.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',18,10),(87,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-361433.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.12.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',22,10),(88,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-362428.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.12.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',26,10),(89,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-363907.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.12.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',30,10),(90,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-445394.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.0.12.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',34,10),(91,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-359781.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.1.0.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',2,11),(92,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-359877.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.1.0.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',6,11),(93,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-360361.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.1.0.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',10,11),(94,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-360457.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.1.0.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',14,11),(95,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-361337.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.1.0.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',18,11),(96,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-361433.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.1.0.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',22,11),(97,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-362428.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.1.0.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',26,11),(98,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-363907.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.1.0.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',30,11),(99,1,'{\"HaplotypeCallerTask.mem_gb\": 32, \"HaplotypeCallerTask.gvcf_mode\": \"false\", \"HaplotypeCallerTask.ref_fasta\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta\", \"HaplotypeCallerTask.input_bams\": [\"NexPond-445394.bam\"], \"HaplotypeCallerTask.gatk_docker\": \"jonnsmith/gatk_test_builds:2019-03-08-4.1.0.0-SNAPSHOT\", \"HaplotypeCallerTask.contamination\": 0, \"HaplotypeCallerTask.interval_list\": \"gs://haplotypecallerspark-evaluation/inputData/whole_exome_illumina_coding_v1.Homo_sapiens_assembly19.targets.interval_list\", \"HaplotypeCallerTask.ref_fasta_dict\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai\", \"HaplotypeCallerTask.ref_fasta_index\": \"gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict\", \"HaplotypeCallerTask.interval_padding\": 0, \"HaplotypeCallerTask.boot_disk_size_gb\": 64, \"HaplotypeCallerTask.default_disk_space_gb\": 512}',34,11);
/*!40000 ALTER TABLE `TestScenario` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `TestScenarioInfo`
--

DROP TABLE IF EXISTS `TestScenarioInfo`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `TestScenarioInfo` (
  `idTestScenarioInfo` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `name` varchar(512) COLLATE utf8mb4_unicode_ci NOT NULL,
  `description` varchar(2048) COLLATE utf8mb4_unicode_ci NOT NULL,
  PRIMARY KEY (`idTestScenarioInfo`),
  UNIQUE KEY `idTestScenarioInfo_UNIQUE` (`idTestScenarioInfo`),
  UNIQUE KEY `name_UNIQUE` (`name`)
) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `TestScenarioInfo`
--

LOCK TABLES `TestScenarioInfo` WRITE;
/*!40000 ALTER TABLE `TestScenarioInfo` DISABLE KEYS */;
INSERT INTO `TestScenarioInfo` VALUES (1,'HaplotypeCaller Validation','Validation tests for the HaplotypeCaller.  Includes both correctness and performance metrics.');
/*!40000 ALTER TABLE `TestScenarioInfo` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `TestScenarioOutputFiles`
--

DROP TABLE IF EXISTS `TestScenarioOutputFiles`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `TestScenarioOutputFiles` (
  `scenarioRun` int(11) unsigned NOT NULL,
  `outputFile` int(11) unsigned NOT NULL,
  PRIMARY KEY (`scenarioRun`,`outputFile`),
  KEY `TestScenarioOutputFiles_outputFileID_idx` (`outputFile`),
  CONSTRAINT `TestScenarioOutputFiles_outputFileID` FOREIGN KEY (`outputFile`) REFERENCES `OutputFiles` (`idOutputFiles`) ON DELETE NO ACTION ON UPDATE NO ACTION,
  CONSTRAINT `TestScenarioOutputFiles_scenarioRunID` FOREIGN KEY (`scenarioRun`) REFERENCES `TestScenario` (`idTestScenarioRun`) ON DELETE NO ACTION ON UPDATE NO ACTION
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `TestScenarioOutputFiles`
--

LOCK TABLES `TestScenarioOutputFiles` WRITE;
/*!40000 ALTER TABLE `TestScenarioOutputFiles` DISABLE KEYS */;
INSERT INTO `TestScenarioOutputFiles` VALUES (1,1),(1,2),(2,3),(2,4),(3,5),(3,6),(4,7),(4,8),(5,9),(5,10),(6,11),(6,12),(7,13),(7,14),(8,15),(8,16),(9,17),(9,18),(10,19),(10,20),(11,21),(11,22),(12,23),(12,24),(13,25),(13,26),(14,27),(14,28),(15,29),(15,30),(16,31),(16,32),(17,33),(17,34),(18,35),(18,36),(19,37),(19,38),(20,39),(20,40),(21,41),(21,42),(22,43),(22,44),(23,45),(23,46),(24,47),(24,48),(25,49),(25,50),(26,51),(26,52),(27,53),(27,54),(28,55),(28,56),(29,57),(29,58),(30,59),(30,60),(31,61),(31,62),(32,63),(32,64),(33,65),(33,66),(34,67),(34,68),(35,69),(35,70),(36,71),(36,72),(37,73),(37,74),(38,75),(38,76),(39,77),(39,78),(40,79),(40,80),(41,81),(41,82),(42,83),(42,84),(43,85),(43,86),(44,87),(44,88),(45,89),(45,90),(46,91),(46,92),(47,93),(47,94),(48,95),(48,96),(49,97),(49,98),(50,99),(50,100),(51,101),(51,102),(52,103),(52,104),(53,105),(53,106),(54,107),(54,108),(55,109),(55,110),(56,111),(56,112),(57,113),(57,114),(58,115),(58,116),(59,117),(59,118),(60,119),(60,120),(61,121),(61,122),(62,123),(62,124),(63,125),(63,126),(64,127),(64,128),(65,129),(65,130),(66,131),(66,132),(67,133),(67,134),(68,135),(68,136),(69,137),(69,138),(70,139),(70,140),(71,141),(71,142),(72,143),(72,144),(73,145),(73,146),(74,147),(74,148),(75,149),(75,150),(76,151),(76,152),(77,153),(77,154),(78,155),(78,156),(79,157),(79,158),(80,159),(80,160),(81,161),(81,162),(82,163),(82,164),(83,165),(83,166),(84,167),(84,168),(85,169),(85,170),(86,171),(86,172),(87,173),(87,174),(88,175),(88,176),(89,177),(89,178),(90,179),(90,180),(91,181),(91,182),(92,183),(92,184),(93,185),(93,186),(94,187),(94,188),(95,189),(95,190),(96,191),(96,192),(97,193),(97,194),(98,195),(98,196),(99,197),(99,198);
/*!40000 ALTER TABLE `TestScenarioOutputFiles` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `Tools`
--

DROP TABLE IF EXISTS `Tools`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `Tools` (
  `idTools` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `version` varchar(1024) COLLATE utf8mb4_unicode_ci NOT NULL,
  `name` varchar(1024) COLLATE utf8mb4_unicode_ci NOT NULL,
  `dateReleased` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
  `wdl` varchar(1024) COLLATE utf8mb4_unicode_ci NOT NULL,
  `wdlChecksum` char(32) COLLATE utf8mb4_unicode_ci NOT NULL,
  PRIMARY KEY (`idTools`),
  UNIQUE KEY `idTools_UNIQUE` (`idTools`)
) ENGINE=InnoDB AUTO_INCREMENT=12 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `Tools`
--

LOCK TABLES `Tools` WRITE;
/*!40000 ALTER TABLE `Tools` DISABLE KEYS */;
INSERT INTO `Tools` VALUES (1,'2019-03-07-4.1.0.0-50-g342569ac0-SNAPSHOT','HaplotypeCaller','2019-03-07 21:21:30','gs://broad-dsp-methods-regression-testing/toolWdls/haplotypeCaller.wdl','30c737cea0697e08eeb79214a068fa97'),(2,'2019-03-08-4.0.4.0-SNAPSHOT','HaplotypeCaller','2018-04-26 13:46:04','gs://broad-dsp-methods-regression-testing/toolWdls/haplotypeCaller.wdl','30c737cea0697e08eeb79214a068fa97'),(3,'2019-03-08-4.0.6.0-SNAPSHOT','HaplotypeCaller','2018-07-06 21:40:31','gs://broad-dsp-methods-regression-testing/toolWdls/haplotypeCaller.wdl','30c737cea0697e08eeb79214a068fa97'),(4,'2019-03-08-4.0.7.0-SNAPSHOT','HaplotypeCaller','2018-07-30 13:46:28','gs://broad-dsp-methods-regression-testing/toolWdls/haplotypeCaller.wdl','30c737cea0697e08eeb79214a068fa97'),(5,'2019-03-08-4.0.8.0-SNAPSHOT','HaplotypeCaller','2018-08-14 18:50:47','gs://broad-dsp-methods-regression-testing/toolWdls/haplotypeCaller.wdl','30c737cea0697e08eeb79214a068fa97'),(6,'2019-03-08-4.0.8.1-SNAPSHOT','HaplotypeCaller','2018-08-16 19:03:52','gs://broad-dsp-methods-regression-testing/toolWdls/haplotypeCaller.wdl','30c737cea0697e08eeb79214a068fa97'),(7,'2019-03-08-4.0.9.0-SNAPSHOT','HaplotypeCaller','2018-09-20 02:27:12','gs://broad-dsp-methods-regression-testing/toolWdls/haplotypeCaller.wdl','30c737cea0697e08eeb79214a068fa97'),(8,'2019-03-08-4.0.10.1-SNAPSHOT','HaplotypeCaller','2018-10-03 20:11:05','gs://broad-dsp-methods-regression-testing/toolWdls/haplotypeCaller.wdl','30c737cea0697e08eeb79214a068fa97'),(9,'2019-03-08-4.0.10.0-SNAPSHOT','HaplotypeCaller','2018-10-09 17:33:58','gs://broad-dsp-methods-regression-testing/toolWdls/haplotypeCaller.wdl','30c737cea0697e08eeb79214a068fa97'),(10,'2019-03-08-4.0.12.0-SNAPSHOT','HaplotypeCaller','2018-12-17 14:47:45','gs://broad-dsp-methods-regression-testing/toolWdls/haplotypeCaller.wdl','30c737cea0697e08eeb79214a068fa97'),(11,'2019-03-08-4.1.0.0-SNAPSHOT','HaplotypeCaller','2019-01-30 02:44:18','gs://broad-dsp-methods-regression-testing/toolWdls/haplotypeCaller.wdl','30c737cea0697e08eeb79214a068fa97');
/*!40000 ALTER TABLE `Tools` ENABLE KEYS */;
UNLOCK TABLES;
/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;

/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;

-- Dump completed on 2019-03-15 15:29:08
